modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/1_application.json

```diff
@@ -305,6 +305,466 @@
+			"email": "jzecchi7@google.cn",
+			"externalReferenceCode": "RAYAPP-42",
+			"firstName": "Jolie",
+			"lastName": "Zecchi",
+			"phone": "559-835-6042",
+			"productCategory": "",
+			"productId": "RAYAP-001",
+			"productName": "Auto",
+			"r_accountToApplications_accountEntryERC": "RAYACCOUNT-003",
+			"state": "CA",
+			"submitDate": "2022-04-14",
+			"user": "",
+			"website": "Null",
+			"zip": "93721"
+		},
+		{
+			"address": "2 Thompson Point",
+			"addressApt": "4",
+			"applicationCreateDate": "2022-02-19",
+			"applicationStatus": "bound",
+			"businessCategoryId": "",
+			"city": "New York City",
+			"dataJSON": "{\"driverInfo\":{\"form\":{\"ageFirstLicensed\":21,\"hasAccidentOrCitations\":\"no\",\"accidentCitation\":null,\"firstName\":\"Pincus\",\"gender\":\"Male\",\"lastName\":\"Middup\",\"maritalStatus\":\"Married\",\"millitaryAffiliation\":null,\"occupation\":\"Dentist\",\"governmentAffliation\":\"ABC Sherriff Association\",\"otherOccupation\":null,\"relationToContact\":\"Self\",\"highestEducation\":\"College\",\"creditRating\":\"Very Good\"}},\"vehicleInfo\":{\"form\":{\"annualMileage\":68314,\"year\":2016,\"make\":\"Toyota\",\"model\":\"Tundra SR6\",\"features\":\"Anti-lock braking system that is factory-installed,Lane-Departure Warning,Blind-Spot Monitoring\",\"ownership\":\"loan\",\"primaryusage\":\"pleasure\"}},\"coverage\":{\"form\":{\"bodilyInjury\":\"$30,000 / $60,000\",\"medical\":\"$2,000\",\"propertyDamage\":\"$300,000\",\"uninsuredOrUnderinsuredMBI\":\"$25,000\",\"uninsuredOrUnderinsuredMPD\":\"$30,000 / $60,000\"},\"vehicles\":{\"collision\":\"$5,000\",\"comprehensive\":\"$750\"}}}",
+			"email": "pmiddup2@ted.com",
+			"externalReferenceCode": "RAYAPP-43",
+			"firstName": "Pincus",
+			"lastName": "Middup",
+			"phone": "212-950-3553",
+			"productCategory": "",
+			"productId": "RAYAP-001",
+			"productName": "Auto",
+			"r_accountToApplications_accountEntryERC": "RAYACCOUNT-004",
+			"state": "NY",
+			"submitDate": "2022-03-05",
+			"user": "",
+			"website": "Null",
+			"zip": "10184"
+		},
+		{
+			"address": "24617 Elmside Parkway",
+			"addressApt": "5",
+			"applicationCreateDate": "2022-08-19",
+			"applicationStatus": "bound",
+			"businessCategoryId": "",
```

Comment:
Please check if `businessCategoryId` and `productCategory` are being used in the flow. If not used, remove it.

Reply:
Ok